index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">Survey-Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+
+    <form action="" id="survey-form">
+
+      <label id="name-label"><b>Name:</b></label>
+            <input id="name" type="name" name="name" class="form" required placeholder="Full Name"><br>
+            <br>
+            <label id="email-label"><b>Email:</b></label>
+            <input id="email" type="email" name="email" class="form" required placeholder="example@mail.com"><br>
+            <br>
+            <label id="number-label"><b>Age:</b></label>
+            <input id="number" type="number" name="number" class="form" min="5" max="99" required placeholder="Age">
+
+
+  
```

Comment:
> Kindly remove the black spaces.

**1-** Please consider always including an explanation, a **"why"**, for each requested change. This means to describe the user why he is being asked for some change.

## General example:-

* **This is a well-explained request:** *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*

* **This request lacks a "why":** *Why don't you use an article tag here instead of a div?*